### DISCUSSION TITLE
Animate Letter inside <code highlight >

### DISCUSSION THREAD

**Discussion by Alexandre-erdnaxelA** [Martinomagnifico/reveal.js-appearance#18] [Q&A] [answered]
Hello !
Is there a way to use “Appearance” to make the text appear letter by letter inside the environment <pre>code class="hljs python languages-python"   /code</pre>

Thanks !

**Comment by Martinomagnifico** [comment, marked as answer]
Hi Alexandre, sorry that won't work. I've looked at loading Highlight first, but while animations of code will work then, the code is no longer highlighted, and carriage returns are also missing.

**Reply by Alexandre-erdnaxelA** [reply]
Thanks for the answer ! That's what i thought...
